src/library/dao/trackdao.cpp

```diff
@@ -1044,7 +1044,7 @@ QList<TrackRef> TrackDAO::getAllTrackRefs(const QDir& rootDir) const {
     }
 
     QList<TrackRef> trackRefs;
-    const int idColumn = query.record().indexOf(LIBRARYTABLE_MIXXXDELETED);
+    const int idColumn = query.record().indexOf(LIBRARYTABLE_ID);
```

Comment:
Fixing a typo likely introduced in https://github.com/mixxxdj/mixxx/commit/9e90d9a2282652446cf21f7c03040662a9d6f115#diff-1778803df6a6852a1b029d65cfee5c4971c163812cea4838859a31fd335f6d0dR1006

Reply:
This was PR #13427